quantus/metrics/robustness/consistency.py

```diff
@@ -62,15 +62,10 @@ def __init__(
             If normalise_func=None, the default value is used, default=normalise_by_max.
         normalise_func_kwargs: dict
             Keyword arguments to be passed to normalise_func on call, default={}.
-        perturb_func: callable
-            Input perturbation function. If None, the default value is used,
-            default=gaussian_noise.
         perturb_std: float
```

Comment:
Please also remove perturb_std and perturb_mean!

Reply:
done